spec/features/checkout_address_selection_spec.rb

```diff
@@ -94,14 +97,12 @@
           fill_in Spree.t(:zipcode), with: '1'
         end
 
-        click_button 'Continue'
```

Comment:
Let's find a way to restore the rest of this test.  This section should still work; it's testing for invalid zip codes, so we don't want to disable it.